dotCMS/src/main/java/com/dotcms/graphql/exception/TypeGenerationException.java

```diff
@@ -0,0 +1,18 @@
+package com.dotcms.graphql.exception;
+
+import com.dotmarketing.exception.DotRuntimeException;
+
+public class TypeGenerationException extends DotRuntimeException {
+
+    public TypeGenerationException(String message) {
+        super(message);
+    }
+
+    public TypeGenerationException(Throwable cause) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'cause' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)